test/extended/prometheus/prometheus.go

```diff
@@ -230,6 +230,44 @@ var _ = g.Describe("[sig-instrumentation] Prometheus", func() {
 				}
 				return true, nil
 			})).NotTo(o.HaveOccurred(), "possibly some services didn't register ServiceMonitors to allow metrics collection")
+
+			g.By("verifying all targets are exposing metrics over secure channel")
+			var insecureTargets []error
+			contents, err := getBearerTokenURLViaPod(ns, execPod.Name, fmt.Sprintf("%s/api/v1/targets", url), bearerToken)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			targets := &prometheusTargets{}
+			err = json.Unmarshal([]byte(contents), targets)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// Currently following targets do not secure their /metrics endpoints:
+			// job="cco-metrics" - https://bugzilla.redhat.com/show_bug.cgi?id=1809194
```

Comment:
Should we leave a note in the bugzillas to remove themselves from this list after they merge their PRs?

Reply:
Yes, I plan on doing that after this is merged.